tasks/linter.py

```diff
@@ -783,3 +784,42 @@ def gitlab_ci_jobs_owners(_, diff_file=None, config_file=None, path_jobowners='.
         )
     else:
         print(f'{color_message("Success", Color.GREEN)}: All jobs have owners defined in {path_jobowners}')
+
+
+@task
+def gitlab_ci_jobs_codeowners(ctx, path_codeowners='.github/CODEOWNERS', all_files=False):
```

Comment:
Is there a reason to focus only on `gitlab` files, and not everything within the git repository?

Reply:
This linter verifies that each job file has a **specific** codeowner rule. In other words, it excludes rules not specific enough such as `.*` or `*` and verifies `/.gitlab/*` patterns. Maybe you are interested in this [PR](https://github.com/DataDog/datadog-agent/pull/29695) for more general usage